### ISSUE TITLE
Railgun "shot" gets more spread out the lower FPS is

### ISSUE THREAD

**Issue by Syncinus** [Anuken/Mindustry#2635]
**Platform**: Windows

**Build**: Custom build, latest commit

**Issue**: Omura railgun shot distance is different based on the FPS (note that in these images, the shot isn't finished firing, so they are different lengths on account of that), this is an image of it at 1000FPS (unlimited): 
![image](https://user-images.githubusercontent.com/34169172/93726370-ebda0980-fb72-11ea-8178-d39eb3223273.png) This is an image of it at 144FPS (VSync): 
![image](https://user-images.githubusercontent.com/34169172/93726386-001e0680-fb73-11ea-8e7e-ec9bf17b9ef5.png)
here is it at 60FPS (normal monitor VSync):
![image](https://user-images.githubusercontent.com/34169172/93726435-365b8600-fb73-11ea-9423-2ff842b85f5f.png)
And finally here is it at 30FPS (normal FPS when game is laggy and stuff):
![image](https://user-images.githubusercontent.com/34169172/93726442-4d9a7380-fb73-11ea-9048-bd509ecff7c7.png)
So basically, the lower the FPS, the more spread out the segments are, it should probably be generating more segments at lower FPS instead of making them more spread out like it does here.

**Steps to reproduce**: Shoot the railgun on the Omura at different FPS's (from graphics setting "Max FPS")

---

*Place an X (no spaces) between the brackets to confirm that you have read the line below.*  
- [X] **I have searched the closed and open issues to make sure that this problem has not already been reported.**

**Comment by Anuken** [comment]
Skipping segments at high FPS is reasonable, but:

> it should probably be generating more segments at lower FPS

The bullet generates one segment per frame, and storing its position every frame and calculate inbetween positions to match the exact distance is both memory intensive and very annoying to implement. Not happening.
